interfaces/apparmor/spec_test.go

```diff
@@ -185,100 +185,164 @@ func (s *specSuite) TestApparmorSnippetsFromLayout(c *C) {
   mount options=(bind, rw) /snap/vanguard/42/foo.conf -> /etc/foo.conf,
   umount /etc/foo.conf,
   # Writable mimic /etc
+  # .. permissions for traversing the prefix that is assumed to exist
+  / r,
+  # .. variant with mimic at /etc/
+  /etc/ rw,
```

Comment:
Regardless actually, if we are without a mimic (direct /etc access on core system with apps running with core as base, aka legacy mode in some other patches) we just write to /etc as it is (though this is where trespassing patches will help). In either case we need /etc/ writable because ... because we write files there, hmm... maybe I'm too sleepy to answer today. 

My point is that we should not assume pivot ever happened.